scripts/sidecar-mysql/entry-point.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+set -xe
+trap '' SIGINT
+trap ''  SIGQUIT
+trap '' SIGTSTP
+
+cd /employees_db
+until mysql -h mysql -u root -p${MYSQL_ROOT_PASSWORD} -t < employees.sql
+do 
+	echo "adding data"
+	sleep 2
+done
+
+while :
```

Comment:
looks like after this, it will only do reads (SELECT).

Let's do a mix of reads and writes in steady state.

Reply:
@thiguetta ^^